lib/rpmi_context.c

```diff
@@ -380,8 +380,7 @@ void rpmi_context_process_a2p_request(struct rpmi_context *cntx)
 			continue;
 
 		rpmi_env_lock(group->lock);
-		if (service && service->process_a2p_request &&
-		    rmsg->header.datalen >= service->min_a2p_request_datalen)
+		if (service && service->process_a2p_request)
```

Comment:
Why drop the check on datalen ?

Reply:
not needed anymore, opensbi was not seniong suspend type before, fixed now.